server/bundler/loader.js

```diff
@@ -16,7 +16,7 @@ function getSectionsModule( sections ) {
 			"\tcontroller = require( 'controller' ),",
 			"\preloadHub = require( 'sections-preload' ).hub;",
```

Comment:
This should be `\tpreloadHub = ...` -- maybe you could fix it with a separate commit while we're here.